test_scripts/API/Navigation/SendLocation/009_SendLocation_boundary.lua

```diff
@@ -27,6 +27,7 @@ local runner = require('user_modules/script_runner')
 local commonSendLocation = require('test_scripts/API/Navigation/commonSendLocation')
 
 --[[ Local Variables ]]
+local maxFileName = string.rep("a", 238)  .. ".png" -- 255 reduced to 242 du to docker limitation
```

Comment:
@dboltovskyi Please fix comment: 
`-- 255 reduced to 242 du to docker limitation` -> `-- 255 reduced to 242 due to docker limitation`

Reply:
@aderiabin: Fixed